lib/i18n/american-english.js

```diff
@@ -335,13 +335,19 @@ function isGoodSentence(sentence) {
         return false;
     if (['.', ',', '?', '!', ' '].includes(sentence[0]))
         return false;
+    // ',' is csv files separator and can cause unwanted sentence splits
+    if (sentence.includes(','))
+        return false;
```

Comment:
Nope. In CSV (unlike TSV), you must a proper CSV library, and then any column with "," will be escaped.

Reply:
fixed.